.ci/docker-run.sh

```diff
@@ -8,7 +8,7 @@ cd .ci
 export BUILDKIT_PROGRESS=plain
 
 if [ "$INTEGRATION" == "true" ]; then
-    docker-compose up --exit-code-from logstash
+    docker compose up --exit-code-from logstash
 else
-    docker-compose up --exit-code-from logstash logstash
+    docker compose up --exit-code-from logstash logstash
```

Comment:
review not: we [changed in upstream .ci](https://github.com/logstash-plugins/.ci/blob/1.x/docker-run.sh#L12), doing same here.